Rasa process starting
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/ubuntu/anaconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Available web server routes: 
static                                             GET, HEAD, OPTIONS             /static/[filename]
custom_webhook_CmdlineInput.health                 GET, HEAD, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                OPTIONS, POST                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5005
Creating a new tracker for id 'default'.
Received user message 'hi' with intent '{'name': 'inform', 'confidence': 0.8816057443618774}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	cuisine: None
	feedback: None
	num_people: None
	outdoor_seating: None
	preferences: None
	requested_slot: None
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'restaurant_form' with prob 0.58.
Calling action endpoint to run action 'restaurant_form'.
Failed to run custom action 'restaurant_form'. Couldn't connect to the server at 'http://localhost:5055/webhook'. Is the server running? Error: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f09b63005c0>: Failed to establish a new connection: [Errno 111] Connection refused',))
Encountered an exception while running action 'restaurant_form'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Failed to execute custom action.
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connection.py", line 141, in _new_conn
    (self.host, self.port), self.timeout, **extra_kw)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py", line 83, in create_connection
    raise err
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py", line 73, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 357, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/home/ubuntu/anaconda3/lib/python3.6/http/client.py", line 1239, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/ubuntu/anaconda3/lib/python3.6/http/client.py", line 1285, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/ubuntu/anaconda3/lib/python3.6/http/client.py", line 1234, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/ubuntu/anaconda3/lib/python3.6/http/client.py", line 1026, in _send_output
    self.send(msg)
  File "/home/ubuntu/anaconda3/lib/python3.6/http/client.py", line 964, in send
    self.connect()
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connection.py", line 166, in connect
    conn = self._new_conn()
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connection.py", line 150, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f09b63005c0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py", line 388, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f09b63005c0>: Failed to establish a new connection: [Errno 111] Connection refused',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/rasa_core/actions/action.py", line 327, in run
    json=json, method="post", timeout=DEFAULT_REQUEST_TIMEOUT)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/rasa_core/utils.py", line 748, in request
    **kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 524, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 637, in send
    r = adapter.send(request, **kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f09b63005c0>: Failed to establish a new connection: [Errno 111] Connection refused',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/rasa_core/processor.py", line 351, in _run_action
    events = action.run(dispatcher, tracker, self.domain)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/rasa_core/actions/action.py", line 348, in run
    raise Exception("Failed to execute custom action.")
Exception: Failed to execute custom action.
Action 'restaurant_form' ended with events '[]'
Current tracker state [None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_slots_values' with prob 0.53.
Action 'utter_slots_values' ended with events '[]'
Bot utterance 'BotUttered(text: got slots:
 - cuisine: None
 - num_people: None
 - outdoor_seating: None
 - preferences: None
 - feedback: None, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'intent_inform': 1.0, 'prev_utter_slots_values': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Rasa process starting
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/ubuntu/anaconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Available web server routes: 
static                                             OPTIONS, HEAD, GET             /static/[filename]
custom_webhook_CmdlineInput.health                 OPTIONS, HEAD, GET             /webhooks/rest/
custom_webhook_CmdlineInput.receive                OPTIONS, POST                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5005
Creating a new tracker for id 'default'.
Received user message 'hi' with intent '{'name': 'inform', 'confidence': 0.8816057443618774}' and entities '[]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	cuisine: None
	feedback: None
	num_people: None
	outdoor_seating: None
	preferences: None
	requested_slot: None
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'restaurant_form' with prob 0.58.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['Form(restaurant_form)', 'SlotSet(key: requested_slot, value: cuisine)']'
Bot utterance 'BotUttered(text: what cuisine?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'i'm looking for an italian restaurant' with intent '{'name': 'request_restaurant', 'confidence': 0.9654631018638611}' and entities '[{'start': 16, 'end': 18, 'value': 'an', 'entity': 'cuisine', 'confidence': 0.6686880591933001, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 8 events
Current slot values: 
	cuisine: None
	feedback: None
	num_people: None
	outdoor_seating: None
	preferences: None
	requested_slot: cuisine
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: cuisine, value: None)', 'SlotSet(key: requested_slot, value: cuisine)']'
Bot utterance 'BotUttered(text: cuisine type is not in the database, please try again, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Bot utterance 'BotUttered(text: what cuisine?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'italian' with intent '{'name': 'inform', 'confidence': 0.8199072480201721}' and entities '[{'start': 0, 'end': 7, 'value': 'italian', 'entity': 'cuisine', 'confidence': 0.9068064236461496, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 15 events
Current slot values: 
	cuisine: None
	feedback: None
	num_people: None
	outdoor_seating: None
	preferences: None
	requested_slot: cuisine
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: cuisine, value: italian)', 'SlotSet(key: requested_slot, value: num_people)']'
Bot utterance 'BotUttered(text: how many people?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message '2' with intent '{'name': 'inform', 'confidence': 0.6989055871963501}' and entities '[{'start': 0, 'end': 1, 'text': '2', 'value': 2, 'confidence': 1.0, 'additional_info': {'value': 2, 'type': 'value'}, 'entity': 'number', 'extractor': 'ner_duckling_http'}]'
Logged UserUtterance - tracker now has 21 events
Current slot values: 
	cuisine: italian
	feedback: None
	num_people: None
	outdoor_seating: None
	preferences: None
	requested_slot: num_people
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: num_people, value: 2)', 'SlotSet(key: requested_slot, value: outdoor_seating)']'
Bot utterance 'BotUttered(text: do you want to seat outside?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'no' with intent '{'name': 'deny', 'confidence': 0.9796501398086548}' and entities '[]'
Logged UserUtterance - tracker now has 27 events
Current slot values: 
	cuisine: italian
	feedback: None
	num_people: 2
	outdoor_seating: None
	preferences: None
	requested_slot: outdoor_seating
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: outdoor_seating, value: False)', 'SlotSet(key: requested_slot, value: preferences)']'
Bot utterance 'BotUttered(text: please provide additional preferences, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'no' with intent '{'name': 'deny', 'confidence': 0.9796501398086548}' and entities '[]'
Logged UserUtterance - tracker now has 33 events
Current slot values: 
	cuisine: italian
	feedback: None
	num_people: 2
	outdoor_seating: False
	preferences: None
	requested_slot: preferences
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: preferences, value: no additional preferences)', 'SlotSet(key: requested_slot, value: feedback)']'
Bot utterance 'BotUttered(text: please give your feedback, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'good' with intent '{'name': 'inform', 'confidence': 0.5431105494499207}' and entities '[]'
Logged UserUtterance - tracker now has 39 events
Current slot values: 
	cuisine: italian
	feedback: None
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: feedback
Current tracker state [None, None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: feedback, value: good)', 'Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Bot utterance 'BotUttered(text: done!, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_slots_values' with prob 0.53.
Action 'utter_slots_values' ended with events '[]'
Bot utterance 'BotUttered(text: got slots:
 - cuisine: italian
 - num_people: 2
 - outdoor_seating: False
 - preferences: no additional preferences
 - feedback: good, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [None, {}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'hi' with intent '{'name': 'inform', 'confidence': 0.8816057443618774}' and entities '[]'
Logged UserUtterance - tracker now has 48 events
Current slot values: 
	cuisine: italian
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_noworries' with prob 0.94.
Action 'utter_noworries' ended with events '[]'
Bot utterance 'BotUttered(text: you are welcome :), data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.97.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'hi' with intent '{'name': 'inform', 'confidence': 0.8816057443618774}' and entities '[]'
Logged UserUtterance - tracker now has 52 events
Current slot values: 
	cuisine: italian
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'restaurant_form' with prob 0.85.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['Form(restaurant_form)', 'Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Bot utterance 'BotUttered(text: done!, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_slots_values' with prob 0.59.
Action 'utter_slots_values' ended with events '[]'
Bot utterance 'BotUttered(text: got slots:
 - cuisine: italian
 - num_people: 2
 - outdoor_seating: False
 - preferences: no additional preferences
 - feedback: good, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'hi' with intent '{'name': 'inform', 'confidence': 0.8816057443618774}' and entities '[]'
Logged UserUtterance - tracker now has 61 events
Current slot values: 
	cuisine: italian
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_noworries' with prob 0.95.
Action 'utter_noworries' ended with events '[]'
Bot utterance 'BotUttered(text: you are welcome :), data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.97.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'i'm looking for asian food' with intent '{'name': 'inform', 'confidence': 0.9192186594009399}' and entities '[{'start': 16, 'end': 21, 'value': 'asian', 'entity': 'cuisine', 'confidence': 0.9948189457525686, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 65 events
Current slot values: 
	cuisine: italian
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'restaurant_form' with prob 0.88.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['Form(restaurant_form)', 'SlotSet(key: cuisine, value: None)', 'SlotSet(key: requested_slot, value: cuisine)']'
Bot utterance 'BotUttered(text: cuisine type is not in the database, please try again, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Bot utterance 'BotUttered(text: what cuisine?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'mexican' with intent '{'name': 'inform', 'confidence': 0.9359993934631348}' and entities '[{'start': 0, 'end': 7, 'value': 'mexican', 'entity': 'cuisine', 'confidence': 0.8977423245676387, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 73 events
Current slot values: 
	cuisine: None
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: cuisine
Current tracker state [{'intent_inform': 1.0, 'prev_restaurant_form': 1.0}, {'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'restaurant_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'restaurant_form' with prob 1.00.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['SlotSet(key: cuisine, value: mexican)', 'Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Bot utterance 'BotUttered(text: done!, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}, {'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0, 'entity_cuisine': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.52.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'for 3 people' with intent '{'name': 'request_restaurant', 'confidence': 0.6624580025672913}' and entities '[{'start': 4, 'end': 5, 'text': '3', 'value': 3, 'confidence': 1.0, 'additional_info': {'value': 3, 'type': 'value'}, 'entity': 'number', 'extractor': 'ner_duckling_http'}]'
Logged UserUtterance - tracker now has 80 events
Current slot values: 
	cuisine: mexican
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'intent_inform': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0, 'entity_cuisine': 1.0}, {'intent_request_restaurant': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_chitchat' with prob 0.57.
Action 'utter_chitchat' ended with events '[]'
Bot utterance 'BotUttered(text: chitchat, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'prev_utter_noworries': 1.0}, {'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0, 'entity_cuisine': 1.0}, {'intent_request_restaurant': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_chitchat': 1.0, 'intent_request_restaurant': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'restaurant_form' with prob 0.91.
Calling action endpoint to run action 'restaurant_form'.
Action 'restaurant_form' ended with events '['Form(restaurant_form)', 'Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Bot utterance 'BotUttered(text: done!, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_inform': 1.0, 'entity_cuisine': 1.0, 'prev_action_listen': 1.0}, {'intent_inform': 1.0, 'prev_restaurant_form': 1.0, 'entity_cuisine': 1.0}, {'intent_request_restaurant': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_chitchat': 1.0, 'intent_request_restaurant': 1.0}, {'intent_request_restaurant': 1.0, 'prev_restaurant_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.67.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'stop' with intent '{'name': 'stop', 'confidence': 0.9778051972389221}' and entities '[]'
Logged UserUtterance - tracker now has 89 events
Current slot values: 
	cuisine: mexican
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'intent_inform': 1.0, 'prev_restaurant_form': 1.0, 'entity_cuisine': 1.0}, {'intent_request_restaurant': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_chitchat': 1.0, 'intent_request_restaurant': 1.0}, {'intent_request_restaurant': 1.0, 'prev_restaurant_form': 1.0}, {'intent_stop': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_ask_continue' with prob 0.98.
Action 'utter_ask_continue' ended with events '[]'
Bot utterance 'BotUttered(text: do you want to continue?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'intent_request_restaurant': 1.0, 'prev_action_listen': 1.0}, {'prev_utter_chitchat': 1.0, 'intent_request_restaurant': 1.0}, {'intent_request_restaurant': 1.0, 'prev_restaurant_form': 1.0}, {'intent_stop': 1.0, 'prev_action_listen': 1.0}, {'intent_stop': 1.0, 'prev_utter_ask_continue': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'no' with intent '{'name': 'deny', 'confidence': 0.9796501398086548}' and entities '[]'
Logged UserUtterance - tracker now has 93 events
Current slot values: 
	cuisine: mexican
	feedback: good
	num_people: 2
	outdoor_seating: False
	preferences: no additional preferences
	requested_slot: None
Current tracker state [{'prev_utter_chitchat': 1.0, 'intent_request_restaurant': 1.0}, {'intent_request_restaurant': 1.0, 'prev_restaurant_form': 1.0}, {'intent_stop': 1.0, 'prev_action_listen': 1.0}, {'intent_stop': 1.0, 'prev_utter_ask_continue': 1.0}, {'intent_deny': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_deactivate_form' with prob 0.74.
Action 'action_deactivate_form' ended with events '['Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Current tracker state [{'intent_request_restaurant': 1.0, 'prev_restaurant_form': 1.0}, {'intent_stop': 1.0, 'prev_action_listen': 1.0}, {'intent_stop': 1.0, 'prev_utter_ask_continue': 1.0}, {'intent_deny': 1.0, 'prev_action_listen': 1.0}, {'intent_deny': 1.0, 'prev_action_deactivate_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Available web server routes: 
static                                             HEAD, GET, OPTIONS             /static/[filename]
custom_webhook_CmdlineInput.health                 HEAD, GET, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Available web server routes: 
static                                             GET, HEAD, OPTIONS             /static/[filename]
custom_webhook_CmdlineInput.health                 GET, HEAD, OPTIONS             /webhooks/rest/
custom_webhook_CmdlineInput.receive                POST, OPTIONS                  /webhooks/rest/webhook
Rasa Core server is up and running on http://localhost:5005
Creating a new tracker for id 'default'.
Received user message 'i'm using aqua stomp' with intent '{'name': 'inform', 'confidence': 0.7863155603408813}' and entities '[{'start': 10, 'end': 20, 'value': 'aqua stomp', 'entity': 'product', 'confidence': 0.9352228531003242, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 2 events
Current slot values: 
	crop: None
	product: None
	requested_slot: None
Current tracker state [None, None, None, {}, {'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'crop_form' with prob 0.95.
Calling action endpoint to run action 'crop_form'.
Action 'crop_form' ended with events '['Form(crop_form)', 'SlotSet(key: product, value: aqua stomp)', 'SlotSet(key: requested_slot, value: crop)']'
Current tracker state [None, None, None, {}, {'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'hi' with intent '{'name': None, 'confidence': 0.0}' and entities '[]'
Logged UserUtterance - tracker now has 8 events
Current slot values: 
	crop: None
	product: aqua stomp
	requested_slot: crop
Current tracker state [None, None, None, {}, {'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'crop_form' with prob 1.00.
Calling action endpoint to run action 'crop_form'.
Failed to validate slot crop with action crop_form
Current tracker state [None, {}, {'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'active_form_crop_form': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_ask_continue' with prob 0.76.
Action 'utter_ask_continue' ended with events '[]'
Bot utterance 'BotUttered(text: do you want to continue?, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{}, {'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'active_form_crop_form': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.99.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'yes' with intent '{'name': None, 'confidence': 0.0}' and entities '[]'
Logged UserUtterance - tracker now has 13 events
Current slot values: 
	crop: None
	product: aqua stomp
	requested_slot: crop
Current tracker state [{'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'active_form_crop_form': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_2_FormPolicy
Predicted next action 'crop_form' with prob 1.00.
Calling action endpoint to run action 'crop_form'.
Failed to validate slot crop with action crop_form
Current tracker state [{'entity_product': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'active_form_crop_form': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}]
There is no memorised next action
There is an active form 'crop_form'
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_deactivate_form' with prob 0.66.
Action 'action_deactivate_form' ended with events '['Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Current tracker state [{'entity_product': 1.0, 'active_form_crop_form': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'prev_action_deactivate_form': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 0.97.
Action 'action_listen' ended with events '[]'
Recreating tracker for id 'default'
Received user message 'i'm using aqua stomp with sonnenblume' with intent '{'name': 'inform', 'confidence': 0.7039792537689209}' and entities '[{'start': 10, 'end': 20, 'value': 'aqua stomp', 'entity': 'product', 'confidence': 0.8791719681939858, 'extractor': 'ner_crf'}, {'start': 26, 'end': 37, 'value': 'sonnenblume', 'entity': 'crop', 'confidence': 0.9454045704751772, 'extractor': 'ner_crf'}]'
Logged UserUtterance - tracker now has 19 events
Current slot values: 
	crop: None
	product: aqua stomp
	requested_slot: None
Current tracker state [{'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'prev_action_deactivate_form': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'crop_form' with prob 0.84.
Calling action endpoint to run action 'crop_form'.
Action 'crop_form' ended with events '['Form(crop_form)', 'SlotSet(key: crop, value: sonnenblume)', 'SlotSet(key: product, value: aqua stomp)', 'Form(None)', 'SlotSet(key: requested_slot, value: None)']'
Bot utterance 'BotUttered(text: here's your details Crop: None , Product: aqua stomp!, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'active_form_crop_form': 1.0, 'prev_utter_ask_continue': 1.0}, {'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'prev_action_deactivate_form': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'utter_slots_values' with prob 0.69.
Action 'utter_slots_values' ended with events '[]'
Bot utterance 'BotUttered(text: got slots:
 - PRODUCT: aqua stomp
 - CROP: sonnenblume
, data: {
  "elements": null,
  "buttons": null,
  "attachment": null
})'
Current tracker state [{'active_form_crop_form': 1.0, 'prev_action_listen': 1.0}, {'prev_action_deactivate_form': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_action_listen': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_crop_form': 1.0, 'intent_inform': 1.0}, {'entity_product': 1.0, 'entity_crop': 1.0, 'prev_utter_slots_values': 1.0, 'intent_inform': 1.0}]
There is no memorised next action
There is no active form
Predicted next action using policy_1_KerasPolicy
Predicted next action 'action_listen' with prob 1.00.
Action 'action_listen' ended with events '[]'
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/rasa_nlu/extractors/entity_synonyms.py:85: UserWarning: Failed to load synonyms file from 'models/nlu/default/current/entity_synonyms.json'
  "".format(entity_synonyms_file))

Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
Rasa process starting
Added 'nlp_spacy' to component cache. Key 'nlp_spacy-en'.
Restoring parameters from models/nlu/default/current/intent_classifier_tensorflow_embedding.ckpt
/home/adil_3.6/miniconda3/lib/python3.6/site-packages/pykwalify/core.py:99: UnsafeLoaderWarning: 
The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
Alternatively include the following in your code:

  import warnings
  warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)

In most other cases you should consider using 'safe_load(stream)'
  data = yaml.load(stream)

Rasa Core server is up and running on http://localhost:5005
